snowfakery/fakedata/fake_data_generator.py

```diff
@@ -26,32 +26,41 @@ def RealisticMaybeRealEmail(self):
 
 
 class FakeData:
-    """Wrapper Faker which adds Salesforce names and case insensitivity."""
+    """Wrapper for Faker which adds Salesforce names and case insensitivity."""
 
     def __init__(self, faker: Faker):
-        self.faker = faker
-        self.fake_names = FakeNames(faker)
+        faker = faker
```

Comment:
?

Reply:
That was "self.faker = faker" and then I did a search and replace to get rid of "self.faker".